### DISCUSSION TITLE
Longhorn fails to be deployed in k8s v.1.25

### DISCUSSION THREAD

**Discussion by t-grzelak** [longhorn/longhorn#4973] [Q&A]
Hello,

couple of weeks ago I deployed Longhorn in k8s v.1.24 without any issues, however it's failing in k8s v1.25. I guess it's due to the removed API for the PodSecurityPolicy (https://kubernetes.io/docs/reference/using-api/deprecation-guide/).
I'm getting the following error:

```
error: resource mapping not found for name: "longhorn-psp" namespace: "" from "longhorn.yaml": no matches for kind "PodSecurityPolicy" in version "extensions/v1beta1"
ensure CRDs are installed first
```

Is it something that could be fixed easily by modifying the yaml manifest file?

**Comment by derekbit** [comment]
LH v1.3.x does not support k8s 1.25.
Please follow up this ticket https://github.com/longhorn/longhorn/issues/4003.

**Reply by t-grzelak** [reply]
Alright, thank you.
Looking forward for the 1.4.0 release!
